src/localization/locales/en-US.json

```diff
@@ -385,7 +384,6 @@
         "form_boundary_options_draw_polygon": "Draw the landscape’s boundary on a map",
         "form_boundary_draw_polygon_edit_help": "<strong>How to edit:</strong> Select a point and drag it to adjust the shape. Select and drag a smaller yellow point to add more points. Remove points using the Delete button. When finished, select {{saveLabel}}.",
         "form_boundary_draw_polygon_edit_help_draw_polygon": "<strong>How to add your boundary:</strong> Select a point on the map at the edge of your landscape. Repeat to make a rough outline of your boundary. When finished, select your first point.",
-        "form_boundary_map_basemap_label": "Background map",
```

Comment:
Issue: It seems that this key is still being used in `OptionAddPin.js`

Reply:
Thanks. Removed it there, too.